src/frontend/efiling-frontend/src/components/page/payment/Payment.js

```diff
@@ -57,23 +57,35 @@ const submitPackage = (submissionId) => {
     .catch((err) => errorRedirect(sessionStorage.getItem("errorUrl"), err));
 };
 
+const checkSubmitEnabled = (paymentAgreed, setSubmitBtnEnabled) => {
+  if (paymentAgreed && sessionStorage.getItem("cardRegistered") === "true") {
+    setSubmitBtnEnabled(true);
+  } else setSubmitBtnEnabled(false);
```

Comment:
```suggestion  
  const isEnabled = paymentAgreed && sessionStorage.getItem("cardRegistered") === "true"
      ? true
      : false;
      
  setSubmitBtnEnabled(isEnabled);
```

something like this you mean? @akroon3r

Reply:
YAR!